torch/testing/_internal/common_nn.py

```diff
@@ -2191,6 +2191,12 @@ def fractional_max_pool3d_test(test_case):
         skip_half=True,
         desc='complex'
     ),
+    dict(
```

Comment:
Should there be an entry that checks complex dtypes like for the 1D and 2D modules?